python/Ice/greeter/server_asyncio/main.py

```diff
@@ -2,29 +2,23 @@
 # Copyright (c) ZeroC, Inc.
 
 import Ice
-import chatbot
 import asyncio
+import chatbot
 import signal
 
 async def main():
 
-    loop = asyncio.get_event_loop()
-
-    def coroutineExecutor(coroutine):
-        """
-        A custom coroutine executor that runs the coroutine in the main asyncio event loop.
-        """
-
-        # We use run_coroutine_threadsafe because the executor is called by an Ice thread.
-        return asyncio.run_coroutine_threadsafe(coroutine, loop)
-
-    # Create an initialization data structure that sets the coroutine executor. The coroutine executor is used by Ice
-    # to run coroutines returned by servant dispatch methods in the main asyncio event loop.
-    initData = Ice.InitializationData()
-    initData.coroutineExecutor = coroutineExecutor
+    # Get the current event loop.
+    loop=asyncio.get_running_loop()
 
-    # Create an Ice communicator to initialize the Ice runtime. The communicator is destroyed at the end of the with block.
-    with Ice.initialize(initData) as communicator:
+    # Create an Ice communicator to initialize the Ice runtime. It will be automatically destroyed at the end of the
+    # with block.
+    #
+    # The event loop is passed to Ice.initialize so that the communicator is configured to use the
```

Comment:
I would use a simpler comment like in the client. No details about future or coroutines. "It just works".